apis/v1alpha3/provider_types.go

```diff
@@ -30,6 +30,10 @@ type ProviderSpec struct {
 
 	// ProjectID is the project name (not numerical ID) of this GCP Provider.
 	ProjectID string `json:"projectID"`
+	// Endpoint overrides the default endpoint.
+	Endpoint string `json:"endpoint"`
+	// WithoutAuthentication - specifies that no authentication should be used. It is suitable only for testing and for accessing public resources.
+	WithoutAuthentication bool `json:"withoutAuthentication"`
```

Comment:
I don't think we need to add this to the `v1alpha3` API as it is already deprecated by `v1beta1`